doc/changelog.md

```diff
@@ -6,35 +6,78 @@ Use a not-yet-used-in-any-PR random line in the top-most version.
 
 Syntax: `- short text describing the change _(Your Name)_`
 
-## 0.3.5 - UNRELEASED
+## 0.3.6 - UNRELEASE
```

Comment:
This PR is a bit of a mixture of things that need to be taken in the release (fix of links, but I think it was already fixed by @tschawax), could be taken in (Cargo.toml changes) and changes that cannot be taken in (pumping of the version number to the next version).
Erlaa
```suggestion
## 0.3.6 - UNRELEASED
```